oneanddone/tasks/filters.py

```diff
@@ -40,7 +40,8 @@ class TasksFilterSet(django_filters.FilterSet):
     search = MultiFieldFilter(
         ['name', 'short_description', 'why_this_matters',
          'prerequisites', 'instructions', 'keyword_set__name'],
-        label=_lazy(u'Search for tasks')
+        label=_lazy(u'Search for tasks'), help_text=_lazy(
```

Comment:
Please start `help_text` on a new line. This will make it consistent with the arguments above it and will make the file more readable.